src/styles/blocks/body.scss

```diff
@@ -0,0 +1,41 @@
+.page {
+  font-size: 18px;
+
+  @media (max-width: 1024px) {
+    font-size: 14px;
+  }
+
+  @media (max-width: 768px) {
+    font-size: 10px;
+  }
+
+  @media (max-height: 980px) {
+    font-size: 10px;
+  }
+
+  @media (max-width: 480px) {
+    font-size: 7px;
+  }
+
+  @media (max-height: 580px) {
+    font-size: 7px;
+  }
+}
+
+.body {
+  & input {
+    position: absolute;
+    top: -9999px;
+  }
```

Comment:
Better create BEM block for this purpose (instead of tag)

Reply:
yes, you right, I'm doing all to not work with tags and better understand BEM princips